migrations/versions/2019_04_18_add_topic_short_title.py

```diff
@@ -0,0 +1,24 @@
+"""empty message
+
+Revision ID: c0dc0cf68e07
+Revises: 2019_04_16_drop_published_bool
+Create Date: 2019-04-18 15:23:51.211497
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'add_topic_short_title'
```

Comment:
And we were going to date-stamp the revision IDs I think to keep migration files in order in the folder (from now on, at least).